edm4hep.yaml

```diff
@@ -141,6 +201,11 @@ components:
         static const int AtCalorimeter = 4 ;\n
         static const int AtVertex = 5 ;\n
         static const int LastLocation = AtVertex  ;\n
+
+        /// Get the covariance matrix value for the two passed parameters\n
+        constexpr float getCovMatrix(edm4hep::TrackParams parI, edm4hep::TrackParams parJ) const { return covMatrix.getValue(parI, parJ); }\n
+        /// Set the covariance matrix value for the two passed parameters\n
+        constexpr void setCovMatrix(float value, edm4hep::TrackParams parI, edm4hep::TrackParams parJ) { covMatrix.setValue(value, parI, parJ); }
```

Comment:
setCovMatrix(std::array const&) missing

Reply:
Shouldn't the `setCovMatrix` be put in `MutableExtraCode` instead of `ExtraCode`?

Reply:
Definitely. Let me check again to have that consistently.

Reply:
After having had another look; The `TrackState` is actually a component. Hence, it only has `ExtraCode` and it doesn't need a `setCovMatrix` because you can just assign to `covMatrix` directly (without having to go through the setter).